### DISCUSSION TITLE
付费后，不能激活专业版，请帮助解决问题

### DISCUSSION THREAD

**Discussion by draercs** [getlantern/lantern-forum-cn#899] [General]
管理员老兄，我已经多次联系你了。也按要求提交了资料。距上次联系又快1个多月过去了，现在还是没有任何进展？？？我的邮箱是（u7tyfui6gf@protonmail.com）。期待你的回复，谢谢。

2024年9月26日 下午 5:40发送 邮件至 mike@getlantern.org 内有付款凭证，

购买时用的账号邮箱u7tyfui6gf@protonmail.com

已经写了好几次电子邮件到（support@getlantern.org） ，但问题照旧，还是免费版。

     2024/02/22 ，在专业版逾期前我就已经续费了617元，续费后变成了免费版。
多次请求解决问题都是机器人回复的，什么问题都没有解决。

    从2024年3月开始，专业版变成免费版，贵公司没有账户体系，邮箱验证码也收不到，半年时间过去了 ，
半年时间无法激活！这6个月我要求能得到补偿。还请尽快来处理这个问题！谢谢

**Comment by draercs** [comment]
是从github上看到您邮箱的，就直接给您发邮件了。请见谅！

希望帮忙给处理提交问题，拜托！！

**Comment by draercs** [comment]
上个月之前已经发帖求助过 ，这个帖子https://github.com/getlantern/lantern-forum-cn/issues/838  ，但是没有得到回复

**Comment by draercs** [comment]
https://lantern.jitbit.com/helpdesk 上提交的工单号 #77375650  ，如下图：

![2024_lantern jitbit com](https://github.com/user-attachments/assets/458ff4c0-8d56-4f95-9eb7-43a68f861b2e)

**Comment by MaxMKMax** [comment]
@draercs 您好. 非常抱歉啊! 您方便的话来GitHub联系我吧. 因为我一般不会查邮箱的. 工作时间有限.

我看到您直接发的邮箱给我了. 然后support@getlantern 是不归我管的. 我没权限. 我就在GitHub处理问题.

像这样新开一个帖子是最好的. 我才会留意到. 谢谢您. 麻烦您了.

好的, 我已经帮您提交问题给支付部门了. 他们会帮您处理. 谢谢您支持蓝灯!
